src/dsymbol/conversion/package.d

```diff
@@ -193,18 +202,29 @@ class SimpleParser : Parser
 				if (moreTokens)
 					advance();
 				if (currentIs(tok!"("))
+				{
+					bool asExpr;
+					if (index < tokens.length - 2 &&
```

Comment:
Given that the only thing inside of the `if` is the `asExpr = true;` line, isn't this better as `immutable bool asExpr = index < tokens.length - 2 && ...`?